docs/components/movement-sensor/gps/gps-nmea-rtk-pmtk.md

```diff
@@ -19,7 +19,7 @@ Breaking changes are likely to occur, and occur often.
 {{% /alert %}}
 
 A global positioning system (GPS) receives signals from satellites in the earth’s orbit to determine where it is and how fast it is going.
-All supported GPS models provide data for the `Position`, `CompassHeading` and `LinearVelocity` methods.
+All supported GPS models provide data for the [`Position`](/components/movement-sensor/#position), [`CompassHeading`](/components/movement-sensor/#compassheading), and [`LinearVelocity`](/components/movement-sensor/#linearvelocity) methods.
 You can obtain fix and correction data by using the sensor `GetReadings` method, which is available because GPSes wrap the [sensor component](/components/sensor/).
```

Comment:
```suggestion
You can obtain fix and correction data by using the sensor [`GetReadings`](/components/sensor/#getreadings) method, which is available because GPSes wrap the [sensor component](/components/sensor/).
```